serving/docs/lmi/configurations_large_model_inference_containers.md

```diff
@@ -3,7 +3,7 @@
 There are a number of shared configurations for python models running large language models.
 They are also available through the [Large Model Inference Containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers).
 
-### Common ([doc](https://github.com/deepjavalibrary/djl-serving/blob/521e0edadec35b04ec9e1d51b9e406119efd0235/serving/docs/configurations_large_model_inference_containers.md#common-doc))
+### Common ([doc](https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-configuration.html))
```

Comment:
also curious, why we are linking awsdocs here since the current github page also contains the same table or possibly more recent parameters as awsdocs updates can take time

Reply:
Yes, AWS docs should refer to this github. This github contains most up-to-date info